compiler/src/codegen/comp_utils.re

```diff
@@ -84,3 +84,7 @@ let load =
   let align = Option.value(~default=sz, align);
   Expression.Load.make(~signed, wasm_mod, sz, offset, align, ty, ptr);
 };
+
+let grain_env_name = "_grainEnv";
+
+let is_grain_env = str => grain_env_name == str;
```

Comment:
Feel free to reject this, but this really spoke to me:
```suggestion
let is_grain_env = (==)(grain_env_name);
```

Reply:
I generally would accept, but in thinking towards Whole Grain, I don't believe that our partial application will look like this, so I'd prefer to keep it just the generic function.